opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -30,10 +30,8 @@
 #include <utility>
 #include <vector>
 
-#include <stddef.h>
```

Comment:
Note that by removing `<stddef.h>`, which I absolutely support, we must also add a namespace qualifier&mdash;i.e., `std::`&mdash;to the current "bare" `size_t` uses in this file.  There is no guarantee that `<cstddef>` makes `size_t` available in the global namespace.

Reply:
fixed.